package.json

```diff
@@ -1,20 +1,14 @@
 {
-  "name": "defillama-apy-server",
```

Comment:
please revert all changes in here, this is not supposed to be updated

Reply:
Reverted